spec/board_spec.rb

```diff
@@ -8,122 +8,77 @@
+  it "should win for each columns" do
+    columns = [[0, 3, 6], [1, 4, 7], [2, 5, 8]]
 
-    expect(board.win?("X")).to be true
-  end
+    columns.each do |array|
+      board = Board.new
 
-  it "should be a win_colum for colum 2" do
-    board.set_mark("X", 1)
-    board.set_mark("X", 4)
-    board.set_mark("X", 7)
+      array.each { |spot| board.set_mark(Mark::CROSS, spot) }
 
-    expect(board.win?("X")).to be true
+      expect(board.win?(Mark::CROSS)).to be true
+    end
   end
 
-  it "should be a win_colum for colum 3" do
-    board.set_mark("X", 2)
-    board.set_mark("X", 5)
-    board.set_mark("X", 8)
+  it "should win for each lines" do
+    lines = [[0, 1, 2], [3, 4, 5], [6, 7, 8]]
 
-    expect(board.win?("X")).to be true
-  end
+    lines.each do |array|
+      board = Board.new
 
-  it "should be a for line 1" do
-    board.set_mark("X", 0)
-    board.set_mark("X", 1)
-    board.set_mark("X", 2)
+      array.each { |spot| board.set_mark(Mark::CROSS, spot) }
 
-    expect(board.win?("X")).to be true
+      expect(board.win?(Mark::CROSS)).to be true
+    end
   end
 
-  it "should be a for line 2" do
-    board.set_mark("X", 3)
-    board.set_mark("X", 4)
-    board.set_mark("X", 5)
+  it "should be a win for diagonal one" do
+    [0, 4, 8].each { |spot| board.set_mark(Mark::CROSS, spot) }
 
-    expect(board.win?("X")).to be true
+    expect(board.win?(Mark::CROSS)).to be true
   end
 
-  it "should be a for line 3" do
-    board.set_mark("X", 6)
-    board.set_mark("X", 7)
-    board.set_mark("X", 8)
-
-    expect(board.win?("X")).to be true
-  end
-
-  it "should be a for diagonal 1" do
-    board.set_mark("X", 0)
-    board.set_mark("X", 4)
-    board.set_mark("X", 8)
-
-    expect(board.win?("X")).to be true
-  end
-  it "should be a for diagonal 2" do
-    board.set_mark("X", 2)
-    board.set_mark("X", 4)
-    board.set_mark("X", 6)
+  it "should be a win for diagonal two" do
+    [2, 4, 6].each { |spot| board.set_mark(Mark::CROSS, spot) }
 
-    expect(board.win?("X")).to be true
+    expect(board.win?(Mark::CROSS)).to be true
   end
 
   it "shoud be a tie" do
-    board.set_mark("X", 0)
-    board.set_mark("O", 1)
-    board.set_mark("X", 2)
-    board.set_mark("X", 3)
-    board.set_mark("O", 4)
-    board.set_mark("X", 5)
-    board.set_mark("O", 6)
-    board.set_mark("X", 7)
-    board.set_mark("O", 8)
+    [0, 2, 3, 5, 7].each { |spot| board.set_mark(Mark::CROSS, spot) }
+    [1, 4, 6, 8].each { |spot| board.set_mark(Mark::ROUND, spot) }
 
     expect(board.tie?).to be true
   end
 
   it "shoud not be a tie" do
-    board.set_mark("X", 0)
-    board.set_mark("X", 1)
-    board.set_mark("X", 2)
-    board.set_mark("X", 3)
-    board.set_mark("O", 4)
-    board.set_mark("X", 5)
-    board.set_mark("O", 6)
-    board.set_mark("X", 7)
-    board.set_mark("O", 8)
+    [0, 1, 2, 3, 5, 7].each { |mark| board.set_mark(Mark::CROSS, mark) }
+    [4, 6, 8].each { |mark| board.set_mark(Mark::CROSS, mark) }
```

Comment:
In all the other tests you use `|spot|` and here you use `|mark|` which is a little confusing since it makes it seem like you're doing something different in this test.  You could abstract this each loop into a method just to make it a little more readable and save you having to type it for every test. Also there's a typo in "should".